haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="https://github.com/grahams/alivejournal"
+COPYRIGHT="2000 Simon Huet
+	2010 Sean Graham"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitSrcRevision="11a4bed0a0800f66ce5069d3df6a0a3cc2be43aa"
+SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="198950a11a0cb38cd9093d08619991ad6bbd2d376e7b1999b09a8a556d9add68"
+SOURCE_FILENAME="alivejournal-$portVersion-$gitSrcRevision.tar.gz"
+SOURCE_DIR="alivejournal-$gitSrcRevision"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	alivejournal = $portVersion
+	app:AliveJournal = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir $docDir
+	install -t $appsDir objects/AliveJournal
+	addAppDeskbarSymlink $appsDir/AliveJournal
+	install -t $docDir README
+	sed -n '/http://www.bebits.com/app/1920/q;p' $docDir/README
+	addResourcesToBinaries $portDir/additional-files/alivejournal.rdef \
```

Comment:
Now that the rdef is available upstream, you can make this change:
```diff
-	addResourcesToBinaries $portDir/additional-files/alivejournal.rdef \
-		$appsDir/AliveJournal
+	addResourcesToBinaries Resource.rdef $appsDir/AliveJournal
```

Oops, I guess you meant "middle = 61" (instead of not "middle = 51") in https://github.com/grahams/alivejournal/commit/fdb75158274cae7b3a96d25c88baf49ba7b8bb32#diff-86a5e8efa2009f917362ed20cb06995eR5

If you create a PR to fix this tiny mistake, maybe you could also see if the Makefile can be changed to automatically process the rdef file to produce and insert the resource in the runtime right after the compilation.
If I understand correctly, makefile_engine has the logic that looks for the RSRCS parameter in the Makefile.
I don't know if it also has the logic to look for a RDEF(S) parameter to process an rdef file.

In case makefile_engine does not know how to process rdef files, then you can either create a tiny PR to fix your typo for the "middle = 61" parameter, or use sed to fix it during the build. But it would be better to create a PR, explaining you made a little typo. I'm sure upstream will kindly merge your second PR.

So if you have the typo fixed, you can then use the simplified addResourcesToBinaries (shown on the top of this comment.)
If not, you will need to fix the middle number before you call addResourcesToBinaries in INSTALL.

Reply:
You removed the PATCH line in the recipe, but you forgot to remove the patch itself.
To do so, you can:
```
cd haiku-apps/alivejournal/patches/
git rm alivejournal-1.61.patchset
git commit alivejournal-1.61.patchset
```